scripts/linters/general_purpose_linter.py

```diff
@@ -210,6 +210,13 @@
         'excluded_files': (),
         'excluded_dirs': ()
     },
+    {
+        'regexp': re.compile(r'urlretrieve\('),
```

Comment:
Will this affect stuff in the core codebase too? I'm not sure if we deploy scripts.common to prod.

Reply:
Fixed to exclude core files